processed_bucket.tf

```diff
@@ -0,0 +1,51 @@
+data "aws_iam_policy_document" "dataworks_aws_collections_rehydration_write" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "s3:GetBucketLocation",
+      "s3:ListBucket",
+    ]
+
+    resources = [
+      data.terraform_remote_state.common.outputs.processed_bucket.arn,
+    ]
+  }
+
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "s3:GetObject*",
+      "s3:DeleteObject*",
+      "s3:PutObject*",
+    ]
+
+    resources = [
+      "${data.terraform_remote_state.common.outputs.processed_bucket.arn}/*"
+    ]
```

Comment:
Seems excessive

Reply:
Does this duplicate functionality in data.tf?

Reply:
Write access removed